src/celt_lpc.c

```diff
@@ -208,7 +208,7 @@ int _celt_autocorr(
                    int          n)
 {
    opus_val32 d;
-   int i, k;
+   int i = 0, k = 0;
```

Comment:
I don't doubt that the static analyzer complains but these initializations are not required because either way both variables are set to 0 when they are used.

Reply:
i think a compromise between initializing everything and being as terse as possible is initializing when a static analyzer highlights a potential problem due to a series of branches, however unlikely that may maybe.

Reply:
I do believe that these are always initialized (and cannot possibly be unintialized), the static analyzer is just not good enough figure out. On the other hand initializing has no drawback either except the mentioned style differences and it is safer in terms of human error. Probably another thing for a maintainer to decide.